tox.ini

```diff
@@ -32,7 +34,7 @@ commands =
 [testenv:docs]
 passenv = *
 deps=
-    Sphinx==1.4.4
+    Sphinx
     sphinx_bootstrap_theme
```

Comment:
Sure, but again, tox is the entry point.  I am trying to promote a single way to do things within the community.